Makefile

```diff
@@ -858,7 +858,9 @@ install.completions:
 .PHONY: install.docker
 install.docker:
 	install ${SELINUXOPT} -d -m 755 $(DESTDIR)$(BINDIR)
-	install ${SELINUXOPT} -m 755 docker $(DESTDIR)$(BINDIR)/docker
+	$(eval COMPILED_DOCKER_SCRIPT := $(shell mktemp))
+	envsubst < docker > ${COMPILED_DOCKER_SCRIPT}
```

Comment:
Requires the `gettext-envsubst` package. What guarantee do we have that this will be installed on a build system? It's on my Fedora laptop, but missing on one of my Debian installations.

Reply:
It's been a default on Ubuntu for ages: `curl -L https://old-releases.ubuntu.com/releases/utopic/ubuntu-14.10-desktop-amd64.manifest | grep 'gettext`. Also seems to be on MacOS by default. It's hard to tell on aggregate though.

If this is a concern, I suggested in the original issue that I could write a binary that uses 
[ExpandEnv](https://pkg.go.dev/os#ExpandEnv) in Go so as to not add another dependency.